GUI/src/Findefusion.cmake

```diff
@@ -25,4 +25,4 @@ set(EFUSION_LIBRARIES ${EFUSION_LIBRARY})
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(EFUSION DEFAULT_MSG EFUSION_LIBRARY EFUSION_INCLUDE_DIR)
-mark_as_advanced(EFUSION_LIBRARY EFUSION_INCLUDE_DIR)
+#mark_as_advanced(EFUSION_LIBRARY EFUSION_INCLUDE_DIR)
```

Comment:
This is only for convenience. cmake on windows is often unable to find paths to libraries and you have to set it manually. This is true also for efusion.lib. If you use cmake-gui, then these two variables get mixed up with other cmake advanced variables and are hidden in the list by default.